### ISSUE TITLE
Suggestion: Add `hashtag` tag to changesets

### ISSUE THREAD

**Issue by amandasaurus** [pietervdvn/MapComplete#1719]
Changes from MapComplete include some hashtags in the changeset source, e.g. [this changeset](https://www.openstreetmap.org/changeset/144506085), however they are added in the `comment` tag of the Changeset,

How about including them in the [`hashtags` changeset tag](https://wiki.openstreetmap.org/wiki/Changeset#Tags_on_changesets) aswell/instead?

**Comment by pietervdvn** [comment]
Hey Amanda,

There are always two hashtags present: one is `#MapComplete` (which is added as `editor`), the other it the name of the theme, which is also added as `theme`. I don't think it is useful to duplicate this information a third time, especially as it is not a human created hashtag - unless there is a tool where having those hashtags would have a benefit?

**Comment by amandasaurus** [comment]
There's a [discussion on the forum](https://community.openstreetmap.org/t/rfc-deprecate-hashtag-only-changeset-comments/105770) about changeset comments only having hashtags. Putting this machine readable data into a changeset tag seems like The Right Thing To Do™.

**Comment by amandasaurus** [comment]
> To be honest: I don't have the time to read through this lengthy forum 
> post. Can you give a summary and indicate the benefit?

tl;dr of the forum: “Changeset comments consisting entirely of `”#hot-task-NNNN, #OrgLovesMaps; #AcmeCorpGivesBack”` aren't very useful for many people”. Mentioned in this thread, is that it's possible to put hashtags as changeset metadata. Hence I suggest it here.

The data nerd in me likes the cleanliness of hashtags being in their own place, and keeping the changeset comment for what people read.

-- 
Amanda

**Comment by pietervdvn** [comment]
The purpose of a changeset comment is to be able to understand the context that a contributor made the change in. As such, another contributor should be able to understand what a contributor _tried_ to do and correct them.

The complaint in the above forum thread is basically that some contributors don't give clear changeset messages but instead give some cryptic hashtags.




MapComplete is in a weird spot here. I try my best to be a good citizen and give as much context to others who analyze a changeset, while keeping the burden on contributors as small as possible by not asking them to write a changeset comment.

This context is readable by having a lot of structured metadata, such as the theme used, the background imagery used, if a point is deleted/moved, by logging (in a structured way) the reason why it was deleted or moved. As such, no cryptic hashtags are really used (except for the theme name in the changeset comment).

As such, I don't feel that adding a 'hashtag'-field which contains duplicate information will add any value here (and rather think it'll only distract).
